addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2614,3 +2614,15 @@ we-button-group.o_grid we-selection-items {
         pointer-events: none;
     }
 }
+
+// OUTDATED SNIPPET ALERT
+we-customizeblock-option.snippet-option-VersionControl {
```

Comment:
Wouldn't be better to add the CSS as:
```CSS
#oe_snippets {
    ...
    > .o_we_customize_panel {
        ...
        we-customizeblock-option {
            ...
            &.snippet-option-VersionControl {...}
```

Instead of a separate CSS rule?

Reply:
I moved it there but I kept it as `we-customizeblock-option.snippet-option-VersionControl`:
- first because there are several `we-customizeblock-option` and my rule did not fit in one of them so I defined it on its own 
- and because there would therefore be a "useless" level of indent if I wrote it with the `&` :wink: